src/jit/emitxarch.cpp

```diff
@@ -11196,9 +11196,21 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
     assert(sz == emitSizeOfInsDsc(id));
 
 #if !FEATURE_FIXED_OUT_ARGS
+    bool bUpdateStackLv = true;
+
+    if (emitIGisInProlog(ig))
+        bUpdateStackLv = false;
+    if (emitIGisInEpilog(ig))
+        bUpdateStackLv = false;
+#if FEATURE_EH_FUNCLETS
+    if (emitIGisInFuncletProlog(ig))
+        bUpdateStackLv = false;
+    if (emitIGisInFuncletEpilog(ig))
+        bUpdateStackLv = false;
+#endif // FEATURE_EH_FUNCLETS
```

Comment:
This is really long piece of code for a simple condition and it seems hard to read. Could you please change it to something like:
```C++
bool bUpdateStackLv =!emitIGisInProlog(ig) && !emitIGisInEpilog(ig);
#if FEATURE_EH_FUNCLETS
bUpdateStackLv &= !emitIGisInFuncletProlog(ig) && !emitIGisInFuncletEpilog(ig);
#endif // FEATURE_EH_FUNCLETS
```
Also, should it be FEATURE_EH_FUNCLETS && defined(_TARGET_X86_) to keep the behavior unchanged for Windows x86?